sleuth_crawler/courses_scraper/courses_scraper/spiders/course_spider.py

```diff
@@ -0,0 +1,40 @@
+import scrapy
+
+class CourseSpider(scrapy.Spider):
+    """Spider that crawls courses.students.ubc.ca
+    For all detailed course data.
+    """
+
+    name = "courses"
+
+    def start_requests(self):
+        urls = [
```

Comment:
I think this was recommended because the there is an option to define a `start_urls` and not implement `start_requests()` at all. I moved the declaration out of the function but kept `start_requests()` just so that it's clear what scrapy is doing in 48d8a46